payment_workflow_enterprise/README.rst

```diff
@@ -0,0 +1,17 @@
+===========================
+Payment Workflow Enterprise
+===========================
+
+This module allows to choose the new contact type Customer Payment if it exists, if it doesn't the invoice address
+is chosen, so that the system set the mails receiver that corresponds to payment orders to the person in charge of them.
+
```

Comment:
@yasmineelmrini this is a technical description of the module. What I meant is a functional description. Developers can read your code and understand.